lib/template/spec/spec_helper.rb

```diff
@@ -10,6 +10,20 @@
 require "bundler"
 Bundler.require(:default, :test)
 
+if ENV["COVERAGE"] || ENV["CI"]
```

Comment:
Mind just adding a short comment here to the same effect as what your pull request description contained? It should help remind us how these get used.